### PR TITLE
1st test & connector

### PR DESCRIPTION
**Pull request by dbulka**
Planes connector with methods
Plane test

### CODE REVIEW

File: aircompany.ini [hunk 1]
```diff
@@ -1,5 +1,5 @@
-[db]
-dbuser=root
-dbpassword=password
+[config]
```
Comment (shukal94) [inline aircompany.ini]: do not commit this file)

File: core/controllers/flight/flights_controller.py [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        :param date_arrival: day of arrival
+        :return: responce of mysql databases
+        """
+        MySQLConnector.INSTANCE.execute_query('use __aircompany;')
+        MySQLConnector.INSTANCE.execute_query('INSERT INTO flights(id_plane, _from, _to, date_departure, date_arrival) '
+                                              'VALUES ({0}, {1}, {2}, {3}, {4});'
+                                              .format(id_plane, _from, _to, date_departure, date_arrival))
+
+
+    def select(self):
```
Comment (shukal94) [inline core/controllers/flight/flights_controller.py]: def read(self) is much more better

File: utils/context.py [hunk 3]
```diff
@@ -2,24 +2,23 @@
 from os import getcwd
 from enum import Enum
 
-CONFIG_FILE_PATH = getcwd() + '/aircompany.ini'
+CONFIG_FILE_PATH = 'C:\\Users\\bulka\\aircompany_repo\\aircompany.ini'
```
Comment (shukal94) [inline utils/context.py:5]: for future commits please remove your own absolute path

File: utils/sandbox.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+def attemp(*args):
+    print('{}'.format(args))
```
Comment (shukal94) [inline utils/sandbox.py]: do not include your test modules into commits

**Review by shukal94 — CHANGES_REQUESTED** [review]
good, but comments

**Review by zipziprar — APPROVED** [review]

**Review by Litsvin — APPROVED** [review]

**Review by shukal94 — APPROVED** [review]

**Pull request merged**